apidef/oas/default.go

```diff
@@ -113,7 +115,7 @@ func (s *OAS) BuildDefaultTykExtension(overRideValues TykExtensionConfigParams,
 		}
 	}
 
-	s.importMiddlewares(overRideValues.AllowList, overRideValues.ValidateRequest)
+	s.importMiddlewares(overRideValues.AllowList, overRideValues.ValidateRequest, overRideValues.MockResponse)
```

Comment:
Maybe let's pass the complete object here, as &overRideValues?

Reply:
It makes sense to pass down complete object since the number of middlewares is increasing, still we have other params for `listenPath`, `customDomain` etc.